src/pages/index.js

```diff
@@ -199,17 +199,105 @@ const Roadmap = () => {
       color: COLORS.green,
       targetQuarter: 'Q2 2020',
     },
+    {
+      title: <Translate>GH Action for Building Apps</Translate>,
+      description: (
+        <Translate>
+          Build your Web application as a Tauri binary for MacOS, Linux and Windows
+        </Translate>
+      ),
+      icon: 'ti-arrow-circle-right',
+      color: COLORS.green,
+      targetQuarter: 'Q3 2020',
+    },
+    {
+      title: <Translate>VS Code Extension</Translate>,
+      description: (
+        <Translate>
+          Commands and validate tauri.conf.json
+        </Translate>
+      ),
+      icon: 'ti-layout-tab',
+      color: COLORS.green,
+      targetQuarter: 'Q3 2020',
+    },
+    {
+      title: <Translate>Core Plugin System</Translate>,
+      description: (
+        <Translate>
+          Update core dependencies automatically from the  CLI.
```

Comment:
wrong description, pasted from the above item :P

Reply:
thanks.